### ISSUE TITLE
[a11y]: Export Modal, Screen reader identifies hidden controls, focus sentinel button

### ISSUE THREAD

**Issue by devadula-nandan** [carbon-design-system/ibm-products#6014]
### Package

@carbon/ibm-products

### Browser

Chrome, Firefox, Edge

### Operating System

Windows

### Package version

v2.49.0-rc.0

### Automated testing tool and ruleset

manual

### Assistive technology

JAWS

### Description

Reported by Raghu during screen reader testing.

Screen reader  identifying - "a  focus sentinel button" on both Top of Modal  which is visually not visible :
1. On Arrow key navigation  this Element identified  both on Top part of the Modal.
2.  "Focus sentinel" Button is not keyboard  operable.
3. Existing Label of this control is not descriptive  enough to understand  the purpose of this Element.

REF# https://github.com/carbon-design-system/carbon/issues/14203

### WCAG 2.1 Violation

1.3.1 Info & Relationships

### Reproduction/example

https://ibm-products.carbondesignsystem.com/?path=/story/ibm-products-components-export-exportmodal--standard&globals=viewport:basic

### Steps to reproduce

This was reported in all variants of the Export Modal.

Navigate Export Modal using either (or both) <kbd>Arrow</kbd> or <kbd>Tab</kbd> keys.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems
